esmvaltool/diag_scripts/perfmetrics/collect.ncl

```diff
@@ -204,12 +396,39 @@ begin
   ; Call plotting function
   if (diag_script_info@metric.eq."taylor") then
     plot = taylor_plot(wks, data_all, "grade")
+  else if (diag_script_info@metric.eq."SMPI") then
+    plot = circle_plot(wks, data_smpi, "performance_index", \
+                       data_smpi@ensemble_name)
   else
     plot = portrait_plot(wks, data_all, "grade")
   end if
+  end if
 
   log_info(" gv " + outfile_plot + "." + file_type)
 
   leave_msg(DIAG_SCRIPT, "")
 
+  ; collect meta-data and call ESMValMD function
+  nc_file = ncdf_outfile
+  plot_file = outfile_plot
+  domain = ("glob")
+  if (diag_script_info@metric.eq."SMPI") then
+    statistics = ("smpi")
+    caption = "Performance index I2. Similar to Figure 1 of Reichler and Kim (2008)."
+    plottype = ("circle")
+    authors = (/"righ_ma", "fran_fr", "gier_be", "hass_bg"/)
+    references = (/"rk2008bams"/)
+    infiles = climofiles
+  else 
+    statistics = ("mean")
+    caption = "Performance metric."
+    plottype = ("taylor")
+    authors = (/"righ_ma", "fran_fr"/)
+    references = (/"righi15gmd", "gleckler08jgr"/)
+    infiles = "will still have to be defined!"     ; TO FIX!
+  end if
+  log_provenance(nc_file, plot_file, caption, statistics, domain, plottype, \
```

Comment:
```suggestion
  log_provenance(nc_file, plot_file, caption, statistics, domains, plottype, \
```
see #951

Reply:
Ok.
These changes also look good to me.